packages/common/container-definitions/src/test/types/validateContainerDefinitionsPrevious.generated.ts

```diff
@@ -384,6 +385,7 @@ declare function get_old_InterfaceDeclaration_IContainerContext():
 declare function use_current_InterfaceDeclaration_IContainerContext(
     use: TypeOnly<current.IContainerContext>);
 use_current_InterfaceDeclaration_IContainerContext(
+    // @ts-expect-error compatibility expected to be broken
```

Comment:
See my other comment about making disposeFn optional.  I believe that will remove the need for this. I believe any "breaking" change to IContainerContext API (forward or backward) is a problem (although it's hard to reason over)